containers/payments/subscription/SubscriptionAddonRow.js

```diff
@@ -0,0 +1,81 @@
+import React, { useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Select, generateUID } from 'react-components';
+import { range } from 'proton-shared/lib/helpers/array';
+import { identity } from 'proton-shared/lib/helpers/function';
+import { c } from 'ttag';
+
+const SubscriptionAddonRow = ({
+    label,
+    price,
+    format = identity,
+    start = 0,
+    min = 0,
+    max = 999,
+    quantity = 0,
+    onChange,
+    step = 1,
+    loading = false
+}) => {
+    const idRef = useRef();
```

Comment:
```suggestion
    const [idRef] = useState(() => generateUID('subscription-addon-row'));
```

To avoid the useEffect